aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -390,6 +390,22 @@ public void observe(
         onObservationFailure.accept(new DataStoreException("Not implemented yet, buster!", "Check back later!"));
     }
 
+    /**
+     * Stops all synchronization processes and deletes the
+     * SQLite database from device's local storage.
```

Comment:
Maybe a note about how/when this stuff is able to resume, afterwards? What if I have some changes in the outbound mutations queue? Will they be sent to AppSync, still?

Reply:
Updated the comment and made a note to come up with a test to verify this.